@kiva/kv-components/vue/stories/KvRadio.stories.js

```diff
@@ -0,0 +1,134 @@
+import KvRadio from '../KvRadio.vue';
+
+export default {
+	title: 'Forms/KvRadio',
+	component: KvRadio,
+	args: {
+		checked: false,
+		disabled: false,
+	},
+};
+
+const Template = (args, {
+	argTypes,
+}) => ({
+	props: Object.keys(argTypes),
+	components: {
+		KvRadio,
+	},
+	data: () => ({
+		radioExampleModel: 'three',
+	}),
+	template: `
```

Comment:
This example seems a little confusing with the model being inside of the label and changing on select. Maybe the example should looks like: 
```
* One radio button
* Two radio button
* Three radio button
* Four radio button

Selected radio value:

Reply:
Agreed, thanks!